motoko/hello-angular-motoko/README.md

```diff
@@ -0,0 +1,82 @@
+    npm run init:dfx
+    ```
+
+-----
+
+## Serving the project locally
+- You can use the following command:
+
+    ```bash
+    # Starts the replica, running in the background
+    # Serves Angular and dfx canisters locally with hot reloading
+    npm run start
+    ```
+    Once the job completes, your application will be available at `http://localhost:4200`.
+
+    If you make any changes while in development mode, the project will recompile on both the Angular and motoko side.
+
+-----
+
+## Deploying the project to the IC:
+From the project root:
+
+ 1. [Follow this guide except for the following](https://kyle-peacock.com/blog/dfinity/your-first-canister) - The `canister_id.json` file you need to modify is in the project root. Update both "motoko" and "www" "ic" properties. Please do not attempt to deploy as provided in the last part of the guide. Instead, run the command in step 2.
+	 
+    ```bash
+    # canister_id.json
+    {
+    "motoko": {
+        # replace this with your unique cansiter from the above guide
+        "ic":"65yca-4yaaa-aaaak-qaqcq-cai"
+    },
+    "www": {
+         # replace this with a second unique cansiter from the above guide
+        "ic": "zcp5t-tqaaa-aaaaj-aijea-cai"
+    }
+   
+    ```
+	
+ 2. Execute `npm run deploy:dfx` to deploy the project to your IC canisters.
+
+CLI commands run from the project root
+
+|         `npm run`       |Description                         
+|----------------|-------------------------------
+|`init:dfx`| Initial bootstrap of the IC.   
+|`start`| Serve both project IC canisters locally with hot reloading.         
+|`deploy:dfx`         |Deploy to IC           
+|`restart:dfx`         |Restarts the local canister execution environment and web server processes. You can use this to Start the IC services as well.
+|`restart:clean:dfx`|Starts the local canister execution environment and web server processes in a clean state by removing checkpoints from your project cache. You can use this to set your project cache to a new state when troubleshooting or debugging.|
+|`generate:motoko`| Generate canister type declarations for motoko supported programming languages. This will generate the `src/declarations` folder and remove conflicts. These generated js files allow you to consume the motoko language service on the front end. i.e `const  ic_service = require('src/declarations/motoko').motoko;`
+
+## Notes
+- The npm `build`, `prebuild` and `prestart` commands are used by the IC processes. Using them may have unintended side effects with other build processes.
+- Currently, the Typescript generated by DFX has trailing commas, which produces a conflict with Angular's default build configurations. Fixing this is beyound the scope of this example.
+- This example is intended to help those already familar with Angular to get started with the IC. This example is designed to be as straight forward as possible but it is not intended as a tool for learning Angular.
```

Comment:
This should not link out to parts of external tutorials. The readme should be self-contained. Can you please simplify this a bit?

Side note: What does not work with `dfx deploy --network ic`? All projects we have so far work with a simple `dfx deploy`.

Reply:
Updated with self-contained Prerequesites. 
Side Note - `dfx deploy --network ic --no-wallet` deploys to the IC (https://ic0.app).  Without it, it deploys locally.